### PR TITLE
Disallow enrolling other users in two-factor authentication

### PR DESCRIPTION
**Pull request by elpete**

### CODE REVIEW

File: modules/contentbox/modules/contentbox-admin/handlers/authors.cfc [hunk 1]
```diff
@@ -332,6 +332,15 @@ component extends="baseHandler"{
         flash.put( "authorID", rc.authorID );
         prc.oAuthor = authorService.get( id=rc.authorID );
 
+        if ( prc.oCurrentAuthor.getAuthorID() != prc.oAuthor.getAuthorID() ) {
+            cbMessagebox.warn( "You cannot enroll another user in two-factor authentication." );
+            flash.keep( "layout,xehInvalidData,xehEnrollmentSuccess,xehEnrollmentSuccessQueryString" )
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/handlers/authors.cfc]: I would just say `flash.keep()` so it keeps all keys.

File: modules/contentbox/modules/contentbox-admin/handlers/authors.cfc [hunk 2]
```diff
@@ -332,6 +332,15 @@ component extends="baseHandler"{
...
+            setNextEvent(
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/handlers/authors.cfc]: Do a hard `return` so when you do the integration test you can test the relocation.

**Comment by elpete** [comment]
@lmajano I was waiting for you to get back to me on that.  What kind of tests are you looking for?  Unit tests seem like spell check here.  An integration test would need a full ContentBox app and database which seems overkill.  Which are you looking for?

**Comment by lmajano** [comment]
I think this should be a simple integration test.  Verifying the relocation.

**Pull request merged**